package.json

```diff
@@ -31,7 +31,7 @@
         "redux-logger": "^3.0.6",
         "redux-thunk": "^2.3.0",
         "reselect": "^4.0.0",
-        "styled-jsx": "3.3.2"
+        "styled-jsx": "^4"
```

Comment:
Were there any issues with styles before this version bump? Just curious about the effects of a possible version mismatch